deploy/kubeturbo-operator/test.sh

```diff
@@ -161,15 +174,47 @@ function uninstall_cr {
 	CR_FILE=$2
 
 	if [ -f "${CR_FILE}" ] && [ -n "${NS}" ]; then
-		kubectl delete -f ${CR_FILE} -n ${NS}
+		$KUBECTL delete -f ${CR_FILE} -n ${NS}
 	fi
 
 	[ -f "${CR_FILE}" ] && rm ${CR_FILE}
-	[ -n "${NS}" ] && kubectl delete ns ${NS}
+	[ -n "${NS}" ] && $KUBECTL delete ns ${NS}
 }
 
+function update_cr_for_logging {
+	CR_NS1=$1
+	CR_FILE1=$2
+	LOGGING_LEVEL=$3
+	echo -e "Updating CR to add logging level..."
+	# existing method: adding spec.args.logginglevel causes pod to restart
+	# sed -i -e "$ a\  args:" $CR_FILE1
+	# sed -i -e "$ a\    logginglevel: 4" $CR_FILE1
+	# new method: adding spec.logging.level should not cause a restart
+	sed -i -e '$ a\  logging:' $CR_FILE1
+	sed -i -e "$ a\    level: $LOGGING_LEVEL" $CR_FILE1	
+	$KUBECTL apply -f ${CR_FILE} -n ${NS}
+}
 
-function main {
+function wait_for_logging_update {
+	NAMESPACE=$1
+	POD_NAME=$2
+	MSG=""
+	declare -i COUNT=0
+
+	while [ -z "$MSG" ]
+	do 
+		echo "Wait for 10s for log level changes to be reflected..."
+		sleep 10
+		MSG=$($KUBECTL -n ${NAMESPACE} logs ${POD_NAME} | grep "Logging level is changed from")
+		if [ $COUNT -eq 10 ]; then
```

Comment:
Maybe changing `10` to `15` is more safe.

Reply:
I am actually doing a loop that checks every 10 secs, so it will wait until that line of log shows up, or time out when it loops 10 times (i.e. 100 seconds).

Reply:
Yes, I know you're doing a 100s waiting here.  I suggested to make a little bigger.  From my experience, sometimes it nearly takes 2mins

Reply:
I see. Usually for me it takes around 40 to 50 secs, but I increased the timeout to 15 just to be safe.